pr_agent/settings/configuration.toml

```diff
@@ -6,6 +6,7 @@ verbosity_level=0 # 0,1,2
 
 [pr_reviewer]
 require_focused_review=true
+require_score_review=true
```

Comment:
It would be beneficial to include comments explaining what each flag does for better readability and maintainability. [medium]